README.md

```diff
@@ -40,7 +40,7 @@ protected void onCreate(Bundle savedInstanceState) {
   super.onCreate(savedInstanceState);
   TileView tileView = new TileView.Builder(this)
   	.setSize(3000, 3000)
-  	.defineDetail("tile-%d-%d.png")
+    .defineZoomLevel("tile-%1$d_%2$d.png")
```

Comment:
whitespace.  also can we use the original template? (less confusing with a single separator instead of two different ones, and the format indices aren't necessary)